tests/integration/run_integration_test.sh

```
@@ -0,0 +1,426 @@
+    log_info "Mounting with httpdirfs (cache mode)..."
+
+    "${HTTPDIRFS_BIN}" \
+        -f \
+        --cache \
+        --cache-location "${CACHE_DIR}" \
+        "${BASE_URL}" \
+        "${CACHE_MOUNT_DIR}" &
+    CACHE_HTTPDIRFS_PID=$!
+
+    # Wait for mount
+    for i in $(seq 1 "${MOUNT_TIMEOUT}"); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)